src/Materials/Textures/renderTargetTexture.ts

```diff
@@ -133,9 +134,20 @@ export class RenderTargetTexture extends Texture {
     public ignoreCameraViewport: boolean = false;
 
     private _postProcessManager: Nullable<PostProcessManager>;
+
+    /**
+     * Post-processes for this render target
+     */
+    public get postProcesses() {
+        return this._postProcesses;
+    }
     private _postProcesses: PostProcess[];
     private _resizeObserver: Nullable<Observer<Engine>>;
 
+    private get _prePass() {

```

Comment:
Maybe add `: boolean` after `private get _prePass()` to make the return type clearer (or rename it to `_prePassEnabled()`?).

Reply:
I like the `_prePassEnabled`